### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by DominikaKarmel**
https://DominikaKarmel.github.io/Museum/

### CODE REVIEW

**Comment by DominikaKarmel** [comment]
wondering why header-background image not uploading to header

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,7 +1,4 @@
-@import "utils";
-@import "fonts";
-@import "typography";
+@import "../styles/utils/reset";
 
-body {
-  background: $c-gray;
-}
+@import "../styles/blocks/title";
+@import "../styles/blocks/header";
```
Comment (anastasiiavorobiova) [inline src/styles/main.scss:4]:
```suggestion
@import "./blocks/header";
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
There should be a relative path for the main.scss file

File: src/styles/main.scss (hunk 1 above)
Comment (DominikaKarmel) [reply src/styles/main.scss:4]: thank You!

**Comment by DominikaKarmel** [comment]
can't figure out why logo 'NAMU" don't wanna be on z-index: 1 position

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
You need to add this picture as a background and then place the logo
image

**Comment by DominikaKarmel** [comment]
i don't know, why I have problem in my main.js
image

**Comment by DominikaKarmel** [comment]
image

**Comment by nazarmatsevych** [comment]
> https://DominikaKarmel.github.io/Museum/

Try to run this command
`npm install @babel/core`

**Comment by nazarmatsevych** [comment]
> and I don't know how I should center the images in the gallery so that it looks good at 320 px wide

Please ask all your questions in fe_chat to get faster response

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Good work, please contact a mentor in fe_chat and resolve all your questions before making re-request

**Comment by DominikaKarmel** [comment]
I have problem with menu, after I've add transform: translateX(100%) menu disapeard. How to solve this problem?

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
To solve this you should add a class to your HTML page Screenshot 2023-10-29 at 16 27 44 and use this type of selector to set your transformation depending on clicked id.
Screenshot 2023-10-29 at 16 26 32
So if we click on the burger - we set translateX(0). If we click on the cross icon (it should have the ID of the body or the first section of your page.

**Comment by DominikaKarmel** [comment]
So: I managed to create a drop-down menu without giving a class for 'html' (on the Polish course they do not recommend giving a class for 'html'). However, the 'burger' icon does not appear. When I go into the developer tools and give 'display: flex' directly to the icon then it appears and everything works normally (you can check). But in general the icon cannot have 'display: flex' or inherit it (it has 'display: static'). Why?

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
If you want to make the icon appear, you should set the non-inline display property to the link. Links are inline elements by default (browsers set this style), so it can cause issues when we put inside them non-inline elements

**Comment by DominikaKarmel** [comment]
> If you want to make the icon appear, you should set the non-inline display property to the link. Links are inline elements by default (browsers set this style), so it can cause issues when we put non-inline elements inside. [More information about this](https://www.geeksforgeeks.org/difference-between-block-elements-and-inline-elements/)

**Comment by DominikaKarmel** [comment]
I have a problem with the resolution on the tablet (768px). In the 'contact-us' section the elements: 'contact-us__opening' and 'contact-us__address' do not want to align to the top edge of the grid. I've tried all possible combinations of alignment of elements in the grid, but I still can't manage it :( help me, please!

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your page
1. Need to remove the scroll when this menu is open
image
Also, the lower part should be darkened as in the design

2. Add a transition for hover effects everywhere
image

3. Also add a hover effects for all clickable elements
image

4. Add scroll-behavior: smooth on the page
for a smooth transition across sections

5. When I click on the button here there are nothing happens, here you need to clear the field when clicking
image

6. These elements should be links
image

**Pull request closed without merging**